src/main/kotlin/org/jetbrains/exposed/dao/UUIDEntity.kt

```diff
@@ -0,0 +1,15 @@
+package org.jetbrains.exposed.dao
+
+import org.jetbrains.exposed.sql.Column
+import java.util.UUID
+
+open class UUIDTable(name: String = "", columName: String = "id") : IdTable<String>(name) {
+  override val id: Column<EntityID<String>> = varchar(columName, 36)
```

Comment:
When using the UUID class, the key is binary and therefore becomes unusable by other ETL tools. Using a 36 character varchar is more appropriate for non originating system interactions.